ui/app/templates/vault/cluster/secrets/backend/configuration.hbs

```diff
@@ -6,7 +6,6 @@
     path="vault.cluster.secrets.backend.list-root"
     model=this.model.id
   }}
-  @isConfigure={{true}}
```

Comment:
`isConfigure` is a parma not used in the SecretListHeader. Somewhere along the way (maybe when the component was glimmerized) this param was leftover. I tripled checked and no functionality changed and the actual param was not mentioned anywhere in the `SecretListHeader` component js or hbs file.